### DISCUSSION TITLE
Are the parameters OK? is local filtering OK?   Thank you!

### DISCUSSION THREAD

**Discussion by sypqys** [tnodir/fort#376] [Q&A]
Hello !

I understand how increase my security ?

![FortFirewall_1sFy3B1pQm](https://github.com/user-attachments/assets/93f91311-2048-459b-b51e-33a5dfe8101f)
![FortFirewall_FV0HmsWgDX](https://github.com/user-attachments/assets/e0df4717-e3ad-423b-adcd-60b695a1f7e3)
![FortFirewall_O0aTxfHBLY](https://github.com/user-attachments/assets/771ce820-b6e3-46ed-a6fc-ec70832c2892)

Thanks !

**Comment by tnodir** [comment]
Yes, it's OK to filter local network addresses.

Can you please use English in screenshots too?

**Reply by sypqys** [reply]
OK sorry

**Comment by sypqys** [comment]
![fort1](https://github.com/user-attachments/assets/65a33319-7868-4006-b617-d15d1acf8264)
![fort2](https://github.com/user-attachments/assets/831795d7-b1df-4517-90df-a8598b96ccb3)
![fort3](https://github.com/user-attachments/assets/6252be39-fb4e-48a5-897e-44cb2cd224cd)

**Comment by sypqys** [comment]
I don't know how to authorize some services and block others?
the user guide I can't see where it is exactly. I speak French.

I saw this on Malwaretips.


Thanks :)

**Reply by tnodir** [reply]
> I don't know how to authorize some services and block others?

Do you mean SvcHost services?

SvcHost services appear in the Programs window having "\svchost\..." path (e.g. `\svchost\w32time`).

Fort can work only with SvcHost services, that run in own separate process. So, you can configure the service in the Programs: Services window, by "Make trackable" button. (It's described in the User Guide and other docs in the Wiki.)

**Reply by sypqys** [reply]
No, I mean that : 
https://malwaretips.com/threads/fort-firewall.113326/post-1107616

"I'm blocking all Windows services and programs, except:
- `DNSCache` service (with "Block Internet", because it works via router and LAN is not filtered)
- `System` (with "Block Internet")
- `BITS`, `DOSvc`, `WuauServ` services for Windows Update, when needed (grouped into App Group)"
I don't know how to blocks all services except those. By making "trackable" ?

**Reply by tnodir** [reply]
1. Make the `DNSCache`, `BITS`, `DOSvc`, `WuauServ` services trackable from Programs: Services
2. Create programs for them by "Add Program" button there
3. Restart computer

`System` is not a service. Try to use `ping.exe` - `System` will appear.

**Reply by sypqys** [reply]
> 1. Make the `DNSCache`, `BITS`, `DOSvc`, `WuauServ` services trackable from Programs: Services
> 
>     2. Create programs for them by "Add Program" button there
> 
>     3. Restart computer
> 
> 
> `System` is not a service. Try to use `ping.exe` - `System` will appear.

Many thanks ! 👍

**Comment by tnodir** [comment]
To increase your security:
- turn on the Options: Self Protection: "Block traffic when Fort Firewall is not running" flag

**Comment by sypqys** [comment]
What is the purpose of the option to block all Internet traffic?

**Reply by tnodir** [reply]
Do you mean "Block Internet Traffic"?
![block-traffic](https://github.com/user-attachments/assets/281f8104-fb63-4c2e-99b3-da2b48b3d378)

It blocks all addresses, except "Local Network Addresses" from Options window.

**Reply by sypqys** [reply]
OK thanks 👍
